src/helpers/RainbowContext.js

```diff
@@ -51,6 +52,7 @@ export default function RainbowContextWrapper({ children }) {
   return (
     <RainbowContext.Provider value={initialValue}>
       {children}
+      {__DEV__ && <DevButton />}
```

Comment:
We use IS_DEV for the dev menu (https://github.com/rainbow-me/rainbow/blob/b969d87acd197a81789bab761bb5eed4345c5244/src/screens/SettingsModal.js#L77) because __DEV__ was giving us some problems with release mode

Reply:
Also, I don't think we want this for iOS or most of us when working on android. Can u hide it via a debug.js variable instead?

Reply:
I agree with bruno, debug seems best. This currently shows up when IS_TESTING is true and I wouldnt want it to interfere